### ISSUE TITLE
Speficy a date for the Internet Archive link

### ISSUE THREAD

**Issue by fcastilloec** [sissbruecker/linkding#637]
I recently decided to use Linkding, mainly because of the Internet Archive functionality. Because I'm coming from a different service, I imported all my bookmarks and now they all have a link in the WaybackMachine.

My problem is that some of my old bookmarks have broken sites. The link to the Internet Archive is pointing to the newest, broken site. I haven't seen a way to edit this link to point it to an earlier version.

For example: [Drunken Irish Stew](http://crepesofwrath.net/2009/12/02/drunken-irish-stew/) returns a page with the body "HTTP Status: 404 (not found)", this is not an error message from the browser but an actual page. The Internet Archive link points to this broken site, but I want to set the URL to [March 23, 2016](https://web.archive.org/web/20160323213809/http://www.thecrepesofwrath.com/2009/12/02/drunken-irish-stew).

Maybe this functionality is already there but I don't know how to do it

**Comment by fcastilloec** [comment]
Actually, I'm going to answer my own question. I just discovered the Admin page, where I can edit every single detail of any bookmark. I was able to change the Internet Archive URL, and even the date displayed for the link.

Given that the functionality is there, I don't think there's a need for a field under the "Edit" menu, especially since not everybody uses the Internet Archive.
Maybe a quick note in your README might help other people find this workaround.
